requirements/pip-compile-all.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+# Run pip-compile to freeze all py3*-requirements.txt files.
+# You should run this every time any top-level requirements in either
+# install-requirements.in or dev-requirements.in are added, removed or changed.
+tox --parallel -e 'py3{6,7,8,9}-requirements'
```

Comment:
it's pretty non-obvious that you have to update this file when py3.10 or w/e comes out.